### DISCUSSION TITLE
On with timed off

### DISCUSSION THREAD

**Discussion by piacco** [Koenkk/zigbee2mqtt#25226] [Devices]
Hello, I've a Girier JR-ZDS01device, with On with timed off functionality. But how do I configure this?
On the Dev Console tab I've filled in the cluster 0x006 with command 0x4001 and payload {"state" : "ON", "on_time": 300} but this gives an error "'genOnOff' has no command '16385"
So could someone explain me what I'am doing wrong?

**Comment by coolspot18** [comment]
It would be nice if this feature were exposed in the UI?
